Small fixes, Ruby tool and Alfred configuration

zaratan opened the pull request.

Commit:
Adding ruby script and alfred integration

denispasin commented on README.md:
```diff
@@ -19,4 +19,29 @@ This script will create Toggl projects with the name `#shotgun_ticket_number sho
+# toggl.rb
 
+## Before first use
+WITH YOUR **SYSTEM** RUBY (`/usr/local/bin/ruby` (for RVM: `rvm use system`)) run: `gem install bundler; bundle install`
```
my system ruby is 2.4.0 and there's a bug in last version of awesome_print published in rubygem...

denispasin commented:
(Je peux pas merge hein, j'ai pas les droits ;) )

Commit:
Adding shortcuts to Alfred

Commit:
When a task has no description it shouldn't fail

Commit:
adds keyring support
adds headless mode so the tool can return a special error code when the credentials can't be found or are out of date.

Commit:
Merge branch 'master' into keyring

Commit:
expired token gets replaced in the .toggl2shotgun file

Commit:
updating gems
